util/index.ts

```diff
@@ -371,55 +385,55 @@ const isDisputeSolver = (community, user) => {
 
 // Return the fee the bot will charge to the seller
 // this fee is a combination from the global bot fee and the community fee
-const getFee = async (amount, communityId) => {
-  const maxFee = Math.round(amount * parseFloat(process.env.MAX_FEE));
+const getFee = async (amount: number, communityId: string) => {
+  const maxFee = Math.round(amount * Number(process.env.MAX_FEE));
   if (!communityId) return maxFee;
 
-  const botFee = maxFee * parseFloat(process.env.FEE_PERCENT);
+  const botFee = maxFee * Number(process.env.FEE_PERCENT);
   let communityFee = Math.round(maxFee - botFee);
   const community = await Community.findOne({ _id: communityId });
   communityFee = communityFee * (community.fee / 100);
 
   return botFee + communityFee;
 };
 
-const itemsFromMessage = str => {
+const itemsFromMessage = (str: string) => {
   return str
     .split(' ')
     .map(e => e.trim())
     .filter(e => !!e);
 };
 
 // Check if a number is int
-const isInt = n => parseInt(n) === n;
+const isInt = (n: number) => Number(n) === n;
```

Comment:
@Mersho this looks like a function that won't be needed when all code is TypeScript